[2015-06-02 01:38:52] local.ERROR: exception 'ErrorException' with message 'Undefined variable: route' in D:\Wamp\www\laravelTest\app\Http\routes.php:27
Stack trace:
#0 D:\Wamp\www\laravelTest\app\Http\routes.php(27): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'D:\\Wamp\\www\\lar...', 27, Array)
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(17158): require('D:\\Wamp\\www\\lar...')
#2 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6819): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(17159): Illuminate\Routing\Router->group(Array, Object(Closure))
#5 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(4858): Illuminate\Container\Container->call(Array)
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(4840): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(17153): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#10 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(1518): Illuminate\Container\Container->call(Array)
#13 D:\Wamp\www\laravelTest\vendor\compiled.php(1510): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#14 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 18)
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(1511): array_walk(Array, Object(Closure))
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(1785): Illuminate\Foundation\Application->boot()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(1311): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(1928): Illuminate\Foundation\Application->bootstrapWith(Array)
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(1890): Illuminate\Foundation\Http\Kernel->bootstrap()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#21 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#22 {main}  
[2015-06-02 01:39:17] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function parameter() on a non-object' in D:\Wamp\www\laravelTest\vendor\compiled.php:7168
Stack trace:
#0 D:\Wamp\www\laravelTest\vendor\compiled.php(1743): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(1738): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 D:\Wamp\www\laravelTest\vendor\compiled.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 D:\Wamp\www\laravelTest\app\Http\routes.php(27): Illuminate\Routing\Router->input()
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(17158): ()
#5 D:\Wamp\www\laravelTest\vendor\compiled.php(6819): App\Providers\RouteServiceProvider->App\Providers\{closure}()
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(6819): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:6819}()
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(17159): Illuminate\Routing\Router->group()
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(925): App\Providers\RouteServiceProvider->map()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array:{D:\Wamp\www\laravelTest\vendor\compiled.php:925}()
#10 D:\Wamp\www\laravelTest\vendor\compiled.php(4858): Illuminate\Container\Container->call()
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(4840): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(17153): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot()
#13 D:\Wamp\www\laravelTest\vendor\compiled.php(925): App\Providers\RouteServiceProvider->boot()
#14 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array:{D:\Wamp\www\laravelTest\vendor\compiled.php:925}()
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(1518): Illuminate\Container\Container->call()
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(1510): Illuminate\Foundation\Application->bootProvider()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(1511): Illuminate\Foundation\Application->Illuminate\Foundation\{closure}()
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(1511): array_walk()
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(1785): Illuminate\Foundation\Application->boot()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(1311): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap()
#21 D:\Wamp\www\laravelTest\vendor\compiled.php(1928): Illuminate\Foundation\Application->bootstrapWith()
#22 D:\Wamp\www\laravelTest\vendor\compiled.php(1890): Illuminate\Foundation\Http\Kernel->bootstrap()
#23 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#24 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#25 D:\Wamp\www\laravelTest\public\index.php(0): {main}()
#26 {main}  
[2015-06-02 01:40:10] local.ERROR: exception 'ErrorException' with message 'Undefined variable: request' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\app\Http\routes.php(29): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'D:\\Wamp\\www\\lar...', 29, Array)
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(17158): require('D:\\Wamp\\www\\lar...')
#2 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6819): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(17159): Illuminate\Routing\Router->group(Array, Object(Closure))
#5 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(4858): Illuminate\Container\Container->call(Array)
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(4840): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(17153): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#10 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(1518): Illuminate\Container\Container->call(Array)
#13 D:\Wamp\www\laravelTest\vendor\compiled.php(1510): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#14 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 18)
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(1511): array_walk(Array, Object(Closure))
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(1785): Illuminate\Foundation\Application->boot()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(1311): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(1928): Illuminate\Foundation\Application->bootstrapWith(Array)
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(1890): Illuminate\Foundation\Http\Kernel->bootstrap()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#21 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#22 {main}  
[2015-06-02 01:40:13] local.ERROR: exception 'ErrorException' with message 'Undefined variable: request' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\app\Http\routes.php(29): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'D:\\Wamp\\www\\lar...', 29, Array)
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(17158): require('D:\\Wamp\\www\\lar...')
#2 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6819): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(17159): Illuminate\Routing\Router->group(Array, Object(Closure))
#5 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(4858): Illuminate\Container\Container->call(Array)
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(4840): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(17153): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#10 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(1518): Illuminate\Container\Container->call(Array)
#13 D:\Wamp\www\laravelTest\vendor\compiled.php(1510): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#14 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 18)
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(1511): array_walk(Array, Object(Closure))
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(1785): Illuminate\Foundation\Application->boot()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(1311): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(1928): Illuminate\Foundation\Application->bootstrapWith(Array)
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(1890): Illuminate\Foundation\Http\Kernel->bootstrap()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#21 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#22 {main}  
[2015-06-02 01:40:18] local.ERROR: exception 'ErrorException' with message 'Undefined variable: request' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\app\Http\routes.php(29): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined varia...', 'D:\\Wamp\\www\\lar...', 29, Array)
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(17158): require('D:\\Wamp\\www\\lar...')
#2 [internal function]: App\Providers\RouteServiceProvider->App\Providers\{closure}(Object(Illuminate\Routing\Router))
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6819): call_user_func(Object(Closure), Object(Illuminate\Routing\Router))
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(17159): Illuminate\Routing\Router->group(Array, Object(Closure))
#5 [internal function]: App\Providers\RouteServiceProvider->map(Object(Illuminate\Routing\Router))
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(4858): Illuminate\Container\Container->call(Array)
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(4840): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(17153): Illuminate\Foundation\Support\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#10 [internal function]: App\Providers\RouteServiceProvider->boot(Object(Illuminate\Routing\Router))
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(925): call_user_func_array(Array, Array)
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(1518): Illuminate\Container\Container->call(Array)
#13 D:\Wamp\www\laravelTest\vendor\compiled.php(1510): Illuminate\Foundation\Application->bootProvider(Object(App\Providers\RouteServiceProvider))
#14 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(App\Providers\RouteServiceProvider), 18)
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(1511): array_walk(Array, Object(Closure))
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(1785): Illuminate\Foundation\Application->boot()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(1311): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(1928): Illuminate\Foundation\Application->bootstrapWith(Array)
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(1890): Illuminate\Foundation\Http\Kernel->bootstrap()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#21 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#22 {main}  
[2015-06-02 02:36:09] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'User' not found' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\vendor\compiled.php(1743): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(1738): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 D:\Wamp\www\laravelTest\vendor\compiled.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): App\Providers\RouteServiceProvider->{closure:D:\Wamp\www\laravelTest\app\Http\routes.php:27-30}()
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:6980}()
#5 D:\Wamp\www\laravelTest\vendor\compiled.php(6973): Illuminate\Routing\Router->performBinding()
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(6967): Illuminate\Routing\Router->substituteBindings()
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(6937): Illuminate\Routing\Router->findRoute()
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch()
#10 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8952}()
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#13 D:\Wamp\www\laravelTest\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle()
#14 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle()
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle()
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
#21 D:\Wamp\www\laravelTest\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#22 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle()
#23 D:\Wamp\www\laravelTest\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#24 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#25 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#26 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8935}()
#27 D:\Wamp\www\laravelTest\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then()
#28 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#29 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#30 D:\Wamp\www\laravelTest\public\index.php(0): {main}()
#31 {main}  
[2015-06-02 02:36:11] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'User' not found' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\vendor\compiled.php(1743): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(1738): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 D:\Wamp\www\laravelTest\vendor\compiled.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): App\Providers\RouteServiceProvider->{closure:D:\Wamp\www\laravelTest\app\Http\routes.php:27-30}()
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:6980}()
#5 D:\Wamp\www\laravelTest\vendor\compiled.php(6973): Illuminate\Routing\Router->performBinding()
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(6967): Illuminate\Routing\Router->substituteBindings()
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(6937): Illuminate\Routing\Router->findRoute()
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch()
#10 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8952}()
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#13 D:\Wamp\www\laravelTest\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle()
#14 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle()
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle()
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
#21 D:\Wamp\www\laravelTest\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#22 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle()
#23 D:\Wamp\www\laravelTest\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#24 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#25 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#26 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8935}()
#27 D:\Wamp\www\laravelTest\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then()
#28 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#29 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#30 D:\Wamp\www\laravelTest\public\index.php(0): {main}()
#31 {main}  
[2015-06-02 02:36:14] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'User' not found' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\vendor\compiled.php(1743): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(1738): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 D:\Wamp\www\laravelTest\vendor\compiled.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): App\Providers\RouteServiceProvider->{closure:D:\Wamp\www\laravelTest\app\Http\routes.php:27-30}()
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:6980}()
#5 D:\Wamp\www\laravelTest\vendor\compiled.php(6973): Illuminate\Routing\Router->performBinding()
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(6967): Illuminate\Routing\Router->substituteBindings()
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(6937): Illuminate\Routing\Router->findRoute()
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch()
#10 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8952}()
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#13 D:\Wamp\www\laravelTest\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle()
#14 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle()
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle()
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
#21 D:\Wamp\www\laravelTest\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#22 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle()
#23 D:\Wamp\www\laravelTest\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#24 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#25 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#26 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8935}()
#27 D:\Wamp\www\laravelTest\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then()
#28 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#29 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#30 D:\Wamp\www\laravelTest\public\index.php(0): {main}()
#31 {main}  
[2015-06-02 02:36:16] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'User' not found' in D:\Wamp\www\laravelTest\app\Http\routes.php:29
Stack trace:
#0 D:\Wamp\www\laravelTest\vendor\compiled.php(1743): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(1738): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 D:\Wamp\www\laravelTest\vendor\compiled.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): App\Providers\RouteServiceProvider->{closure:D:\Wamp\www\laravelTest\app\Http\routes.php:27-30}()
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(6980): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:6980}()
#5 D:\Wamp\www\laravelTest\vendor\compiled.php(6973): Illuminate\Routing\Router->performBinding()
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(6967): Illuminate\Routing\Router->substituteBindings()
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(6937): Illuminate\Routing\Router->findRoute()
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch()
#10 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8952}()
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#13 D:\Wamp\www\laravelTest\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle()
#14 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle()
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle()
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
#21 D:\Wamp\www\laravelTest\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#22 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle()
#23 D:\Wamp\www\laravelTest\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#24 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#25 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#26 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8935}()
#27 D:\Wamp\www\laravelTest\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then()
#28 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#29 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#30 D:\Wamp\www\laravelTest\public\index.php(0): {main}()
#31 {main}  
[2015-06-02 02:42:22] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in D:\Wamp\www\laravelTest\storage\framework\views\e8f09ba57584582a82c77b1c838e65c3:9
Stack trace:
#0 D:\Wamp\www\laravelTest\vendor\compiled.php(1743): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 D:\Wamp\www\laravelTest\vendor\compiled.php(1738): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 D:\Wamp\www\laravelTest\vendor\compiled.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 D:\Wamp\www\laravelTest\vendor\compiled.php(13683): Illuminate\View\Engines\PhpEngine->evaluatePath()
#4 D:\Wamp\www\laravelTest\vendor\compiled.php(13523): Illuminate\View\Engines\CompilerEngine->get()
#5 D:\Wamp\www\laravelTest\vendor\compiled.php(13510): Illuminate\View\View->getContents()
#6 D:\Wamp\www\laravelTest\vendor\compiled.php(13501): Illuminate\View\View->renderContents()
#7 D:\Wamp\www\laravelTest\vendor\compiled.php(14597): Illuminate\View\View->render()
#8 D:\Wamp\www\laravelTest\vendor\compiled.php(14126): Illuminate\Http\Response->setContent()
#9 D:\Wamp\www\laravelTest\vendor\compiled.php(7154): Symfony\Component\HttpFoundation\Response->__construct()
#10 D:\Wamp\www\laravelTest\vendor\compiled.php(6954): Illuminate\Routing\Router->prepareResponse()
#11 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#12 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8952}()
#13 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#14 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8935}()
#15 D:\Wamp\www\laravelTest\vendor\compiled.php(6955): Illuminate\Pipeline\Pipeline->then()
#16 D:\Wamp\www\laravelTest\vendor\compiled.php(6944): Illuminate\Routing\Router->runRouteWithinStack()
#17 D:\Wamp\www\laravelTest\vendor\compiled.php(6929): Illuminate\Routing\Router->dispatchToRoute()
#18 D:\Wamp\www\laravelTest\vendor\compiled.php(1935): Illuminate\Routing\Router->dispatch()
#19 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
#20 D:\Wamp\www\laravelTest\vendor\compiled.php(8952): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8952}()
#21 D:\Wamp\www\laravelTest\vendor\compiled.php(2438): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#22 D:\Wamp\www\laravelTest\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle()
#23 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): App\Http\Middleware\VerifyCsrfToken->handle()
#24 D:\Wamp\www\laravelTest\vendor\compiled.php(12083): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#25 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\View\Middleware\ShareErrorsFromSession->handle()
#26 D:\Wamp\www\laravelTest\vendor\compiled.php(10785): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#27 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Session\Middleware\StartSession->handle()
#28 D:\Wamp\www\laravelTest\vendor\compiled.php(11789): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#29 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
#30 D:\Wamp\www\laravelTest\vendor\compiled.php(11738): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#31 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Cookie\Middleware\EncryptCookies->handle()
#32 D:\Wamp\www\laravelTest\vendor\compiled.php(2478): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#33 D:\Wamp\www\laravelTest\vendor\compiled.php(8944): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#34 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#35 D:\Wamp\www\laravelTest\vendor\compiled.php(8935): call_user_func:{D:\Wamp\www\laravelTest\vendor\compiled.php:8935}()
#36 D:\Wamp\www\laravelTest\vendor\compiled.php(1891): Illuminate\Pipeline\Pipeline->then()
#37 D:\Wamp\www\laravelTest\vendor\compiled.php(1880): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#38 D:\Wamp\www\laravelTest\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle()
#39 D:\Wamp\www\laravelTest\public\index.php(0): {main}()
#40 {main}  
